src_code/express_app/app.js

```diff
@@ -12,13 +12,15 @@ app.use(express.json());
 /** Import routes */
 const authenticationRoutes = require('../src/routes/AuthenticationRoutes');
 const inventoryRoutes = require('../src/routes/InventoryRoutes');
+const reservationRoutes = require('../src/routes/ReservationRoutes');
 
 /** Define the port */
 const PORT = 5000;
```

Comment:
Prob a small thing, but I've noticed that there's some inconsistency with the port nums, at least in the dev environment. Here the port is 5000, but the dev_url variable in Postman is 5001. I personally think it might be best to add the other port in, just as a comment with a TODO reminder to delete it afterwards.